tests/integration/session/test_login.py

```diff
@@ -287,6 +299,16 @@ def test_login_with_invalid_questionnaire_claims_should_be_forbidden(self):
 
         self.assertStatusForbidden()
 
+    def test_login_with_invalid_questionnaire_claims_should_be_forbidden_v2(self):
```

Comment:
I am not sure about tests having same names (even worse, in the same module), but they are part of different classes, but still, should be something like `test_login_with_invalid_questionnaire_claims_should_be_forbidden_v2_post`.
But I know it wasn't your design so not your fault, I don't expect you to change it, so just a note.

Reply:
I'm happy to change this as it may be difficult for someone to navigate to find the correct tests

Reply:
Yeah, that's what I was thinking, you will be send to the wrong method when traversing. Maybe lets wait for some other devs opinion on this?